analyzer/src/funTest/assets/projects/synthetic/spdx-project-xyz-expected-output.yml

```diff
@@ -12,12 +12,12 @@ project:
     type: "Git"
     url: "<REPLACE_URL_PROCESSED>"
     revision: "<REPLACE_REVISION>"
-    path: "analyzer/src/funTest/assets/projects/synthetic/spdx"
+    path: "<REPLACE_PATH>"
```

Comment:
Nit: The commit message could make clearer that this does not move the whole projects, but only the definition (SPDX) files. Because when reading I first thought this would somewhat undo the directory merge done in 77a2dc062da5f31680f9278d1e7044baca3af80b, which I'd like to keep.

Reply:
The commit message has been slightly reworded.